src/testflinger_device_connectors/devices/oemrecovery/oemrecovery.py

```diff
@@ -121,11 +185,14 @@ def check_device_booted(self):
         # Wait for provisioning to complete - can take a very long time
         while time.time() - started < 3600:
```

Comment:
Because this was already here, we really end up with a timeout within a timeout. Perhaps it could be simplified?